### DISCUSSION TITLE
missing code indentation in PR view

### DISCUSSION THREAD

**Discussion by casperdcl** [community/community#55836] [Mobile] [answered]
Leading whitespace is stripped.

Tested on GitHub Mobile (Android) on a PR altering Python code. Tried with & without code wrapping enabled.

**Comment by aconsuegra** [comment, marked as answer]
Hi @casperdcl, thanks for reaching out. We found the bug at the beginning of the week and shipped a fix that it is already in our Beta build. If you want to verify the fix, you can join the Beta testers rogram in Google Play, otherwise the fix will be available in our Production build next week.

**Reply by casperdcl** [reply]
Excellent, thanks!
